### ISSUE TITLE
ValidationAdmissionPolicy binding example is incorrect

### ISSUE THREAD

**Issue by samos123** [kubernetes/website#38495]
**Problem:**
The following example is incorrect:
```
apiVersion: admissionregistration.k8s.io/v1alpha1
kind: ValidatingAdmissionPolicyBinding
metadata:
  name: "demo-binding-test.example.com"
spec:
  policy: "replicalimit-policy.example.com"
  matchResources:
    namespaceSelectors:
    - key: environment,
      operator: In,
      values: ["test"]
```
There is no field `policy` and there is no field `namespaceSelectors` under `matchResources`.  See the API spec here for reference: https://pkg.go.dev/k8s.io/api/admissionregistration/v1alpha1#ValidatingAdmissionPolicyBindingSpec

**Proposed Solution:**
Simplify the examples on the page to be same or similar to https://samos-it.com/posts/trying-cel-admission-control-validation-admission-policy.html

So for example use this instead:
```
apiVersion: admissionregistration.k8s.io/v1alpha1
kind: ValidatingAdmissionPolicyBinding
metadata:
  name: "demo-binding-test.example.com"
spec:
  policyName: "demo-policy.example.com"
  matchResources:
    namespaceSelector:
      matchLabels:
        environment: test
```

**Page to Update:**
https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/

In addition, I don't think adding `.example.com` adds any value. Maybe we can remove that as well?

**Comment by samos123** [comment]
Feel free to assign to me

**Comment by sftim** [comment]
/language en

Anyone who [signs the CLA](https://www.kubernetes.dev/docs/guide/#sign-the-cla) and abides by our project code of conduct is welcome to open a PR to fix this issue; Kubernetes is open source.

**Comment by sftim** [comment]
If any of these examples are things people might reasonably deploy to a real cluster, consider switching to a `codenew` shortcode - see https://kubernetes.io/docs/contribute/style/write-new-topic/#embedding-code-in-your-topic

**Comment by jihoon-seo** [comment]
This issue seems to be resolved by the PR #38497.
/assign @samos123

**Comment by tengqm** [comment]
Feel free to reopen if the problem is not resolved.
/close
